### ISSUE TITLE
Why softmax classification loss function is convex

### ISSUE THREAD

**Issue by YerongLeopard** [YerongLeopard/CNN-cs231n_2016_Stanford#2]
In the reference "Softmax Regression" , it says, J(\theta) is still convex
